File: app/src/main/java/com/mushuichuan/removebadge/MainActivity.java
package com.mushuichuan.removebadge;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        findViewById(R.id.samsung).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Util.sendToSamsumg(MainActivity.this, getPackageName(), MainActivity.this.getComponentName().getClassName(),3);
            }
        });

    }

}

File: app/src/main/java/com/mushuichuan/removebadge/MainActivity.kt
package com.mushuichuan.removebadge

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.Toolbar

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val toolbar = findViewById(R.id.toolbar) as Toolbar?
        setSupportActionBar(toolbar)
        var num=1;
        findViewById(R.id.samsung)!!.setOnClickListener { Util.sendToOneApp(this@MainActivity, packageName, this@MainActivity.componentName.className, num++) }
        findViewById(R.id.samsung_add_all)!!.setOnClickListener { Util.sendToAll(this@MainActivity, 100) }
    }

}
